src/metabase/api/public.clj

```diff
@@ -308,7 +309,7 @@
 (api/defendpoint POST ["/dashboard/:uuid/dashcard/:dashcard-id/card/:card-id/:export-format"
                        :export-format api.dataset/export-format-regex]
   "Fetch the results of running a publicly-accessible Card belonging to a Dashboard and return the data in one of the export formats. Does not require auth credentials. Public sharing must be enabled."
-  [uuid card-id dashcard-id parameters export-format]
+  [uuid card-id dashcard-id parameters export-format :as {{:keys [format-rows pivot-results]} :body}]
```

Comment:
I see in some places it is `format_rows` but in others `format-rows`. Will BE accept the underscore variant of the parameter in every endpoint?

Reply:
I'll double check, but in the POST body it should be ok to send as `format_rows`